tests/CommonTests.cs

```diff
@@ -380,44 +379,51 @@ public async Task CanHandleKeysend()
                 var info = await dest.GetInfo();
                 var node = info.NodeInfoList.First();
                 
-                var amount = LightMoney.Satoshis(21);
-                var preimage = new byte[32];
-                new Random().NextBytes(preimage);
-                var paymentHash = new uint256(Hashes.SHA256(preimage));
+                var amount = LightMoney.Satoshis(6);
 
                 // https://github.com/satoshisstream/satoshis.stream/blob/main/TLV_registry.md
-                var val5482373484 = Encoders.Base64.EncodeData(preimage);
-                var val696969 = Encoders.Base64.EncodeData(Encoding.Default.GetBytes("123456"));
-                var val112111100 = Encoders.Base64.EncodeData(Encoding.Default.GetBytes("wal_hrDHs0RBEM576"));
-                var tlvData = new Dictionary<ulong,string>
+                var val696969 = Encoding.Default.GetBytes("123456");
+                var val112111100 = Encoding.Default.GetBytes("wal_hrDHs0RBEM576");
+                var tlvData = new Dictionary<ulong, byte[]>
```

Comment:
Pass in the values as `byte[]` and also get them back as `byte[]`, so that we can support it across implementations.